oam/post/CMakeLists.txt

```diff
@@ -1,3 +1,7 @@
+configure_file("${CMAKE_CURRENT_SOURCE_DIR}/mcstest-001.sh.in" "${CMAKE_CURRENT_SOURCE_DIR}/mcstest-001.sh" @ONLY)
```

Comment:
Do we really need these files?

Reply:
Apparently, yes. They appear to be used by ProcMon to make sure the node is really up.